packages/ui-deprecated/components/ui/segmented-picker/index.tsx

```diff
@@ -83,7 +84,7 @@ export const SegmentedPicker = <ValueType extends { toString: () => string }>({
             size === 'md' && 'text-sm px-3 h-8',
             size === 'lg' && 'px-4 h-10',
             value !== option.value && 'text-light-grey',
-            grow && 'grow',
+            grow && 'flex-1', // Use 'flex-1' for equal growth
```

Comment:
Needed to add this as the tabs would be determined by text width and not be even